proto_lldp.c

```diff
@@ -99,13 +99,21 @@ static int lldp_print_net_addr(const uint8_t *addr, size_t addrlen)
 	case IANA_AF_IPV4:
 		if (addrlen < 4)
 			return -EINVAL;
-		inet_ntop(AF_INET, addr, buf, sizeof(buf));
+		if (NULL == inet_ntop(AF_INET, addr, buf, sizeof(buf))) {
```

Comment:
Please change this to `inet_ntop(...) == NULL`, this is the format for return value checks we use in netsniff-ng.